lib/screen/profile_screen.dart

```diff
@@ -5,6 +5,7 @@ import '../widgets/my_navigation_bar.dart';
 import '../widgets/outlined_button.dart';
 
 class ProfileScreen extends StatelessWidget {
+  final bool group = false;
   const ProfileScreen({Key? key}) : super(key: key);
```

Comment:
`group` should be passed in as a required parameter to the ProfileScreen constructor, so that wherever a ProfileScreen is created, it knows which screen to render
also, I think that renaming the param to `isGroup` would be slightly clearer!